services/ui-src/src/utils/forms/forms.ts

```diff
@@ -228,3 +231,80 @@ export const resetClearProp = (fields: (FormField | FormLayoutElement)[]) => {
     }
   });
 };
+
+export const convertEntityToTargetPopulationChoice = (
+  entity: EntityShape[]
+) => {
+  return entity?.map((field: EntityShape) => {
+    return {
+      key: `targetPopulations-${field.id}`,
+      value: field.isRequired
+        ? field.transitionBenchmarks_targetPopulationName
+        : `Other: ${field.transitionBenchmarks_targetPopulationName}`,
+    };
+  });
+};
+
+export const convertChoiceToEntity = (choices: Choice[]) => {
```

Comment:
non-blocking: would be super helpful for the future if you left a quick comment defining what the difference between the following 2 functions are and when to use each (`convertChoiceToEntity`, and `convertTargetPopulationsFromWPToSAREntity`)

Reply:
Done :D